### ISSUE TITLE
Container newer versions crashing on startup

### ISSUE THREAD

**Issue by RealStickman** [gotson/komga#904]
### Steps to reproduce

The newer images 0.154.2 and 0.154.3 seem to want more RAM than previous images.  
Previous versions until 0.154.1 worked fine in the exact same environment.  

I'm running the container on a debian host, previously with 4gb RAM and now with 6gb, though it still does not start.

When starting the container, it will immediately exit again and throw an insufficient memory error.

### Expected behavior

The container should start normally

### Actual behavior

Container exits/crashes right after startup  
Notably, the container does not get far enough to dump logs into `/config/logs`, so I can only show the container log and a log contained in `/app` within the container.

### Logs

<details><summary>Container logs:</summary>
<p>

```
[0.002s][warning][os,thread] Failed to start thread - pthread_create failed (EPERM) for attributes: stacksize: 1024k, guardsize: 4k, detached.
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Cannot create worker GC thread. Out of system resources.
# An error report file with more information is saved as:
# /app/hs_err_pid1.log
```
</p>
</details>

<details><summary>App log mentioned above:</summary>
<p>

```
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Cannot create worker GC thread. Out of system resources.
# Possible reasons:
#   The system is out of physical RAM or swap space
#   The process is running with CompressedOops enabled, and the Java Heap may be blocking the growth of the native heap
# Possible solutions:
#   Reduce memory load on the system
#   Increase physical memory or swap space
#   Check if swap backing store is full
#   Decrease Java heap size (-Xmx/-Xms)
#   Decrease number of Java threads
#   Decrease Java thread stack sizes (-Xss)
#   Set larger code cache with -XX:ReservedCodeCacheSize=
#   JVM is running with Unscaled Compressed Oops mode in which the Java heap is
#     placed in the first 4GB address space. The Java Heap base address is the
#     maximum limit for the native heap growth. Please use -XX:HeapBaseMinAddress
#     to set the Java Heap base and to place the Java Heap above 4GB virtual address.
# This output file may be truncated or incomplete.
#
#  Out of Memory Error (workerManager.hpp:70), pid=1, tid=1
#
# JRE version:  (11.0.15+10) (build )
# Java VM: OpenJDK 64-Bit Server VM (11.0.15+10, mixed mode, sharing, tiered, compressed oops, g1 gc, linux-amd64)
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#

---------------  S U M M A R Y ------------

Command Line: org.springframework.boot.loader.JarLauncher --spring.config.additional-location=file:/config/

Host: AMD Ryzen 5 3600 6-Core Processor, 2 cores, 5G, Ubuntu 22.04 LTS
Time: Sun Jun 12 12:55:14 2022 UTC elapsed time: 0.002942 seconds (0d 0h 0m 0s)

---------------  T H R E A D  ---------------

Current thread (0x0000555c45b8d000):  JavaThread "Unknown thread" [_thread_in_vm, id=1, stack(0x00007fff3f2fe000,0x00007fff3fafd000)]

Stack: [0x00007fff3f2fe000,0x00007fff3fafd000],  sp=0x00007fff3faf72d0,  free space=8164k
Native frames: (J=compiled Java code, A=aot compiled Java code, j=interpreted, Vv=VM code, C=native code)
V  [libjvm.so+0xf424ca]  VMError::report_and_die(int, char const*, char const*, __va_list_tag*, Thread*, unsigned char*, void*, void*, char const*, int, unsigned long)+0x1ca
V  [libjvm.so+0xf4338b]  VMError::report_and_die(Thread*, char const*, int, unsigned long, VMErrorType, char const*, __va_list_tag*)+0x2b
V  [libjvm.so+0x6cf005]  report_vm_out_of_memory(char const*, int, unsigned long, VMErrorType, char const*, ...)+0xd5
V  [libjvm.so+0xf6f58b]  AbstractWorkGang::add_workers(unsigned int, bool)+0x14b
V  [libjvm.so+0x7aa8c5]  G1CollectedHeap::G1CollectedHeap(G1CollectorPolicy*)+0x5b5
V  [libjvm.so+0x79c3b9]  G1Arguments::create_heap()+0x69
V  [libjvm.so+0xefdc11]  Universe::initialize_heap()+0x31
V  [libjvm.so+0xefdec6]  universe_init()+0x46
V  [libjvm.so+0x8a4207]  init_globals()+0x57
V  [libjvm.so+0xed5537]  Threads::create_vm(JavaVMInitArgs*, bool*)+0x357
V  [libjvm.so+0x960262]  JNI_CreateJavaVM+0x52
C  [libjli.so+0x49af]  JavaMain+0x8f
C  [libjli.so+0x9d94]  CallJavaMainInNewThread+0x74
C  [libjli.so+0x637d]  ContinueInNewThread+0x4d
C  [libjli.so+0x7a00]  JLI_Launch+0x1470
C  [java+0xa73]


---------------  P R O C E S S  ---------------

Threads class SMR info:
_java_thread_list=0x0000555c45b55580, length=0, elements={
}

Java Threads: ( => current thread )

Other Threads:

=>0x0000555c45b8d000 (exited) JavaThread "Unknown thread" [_thread_in_vm, id=1, stack(0x00007fff3f2fe000,0x00007fff3fafd000)]

Threads with active compile tasks:

VM state:not at safepoint (not fully initialized)

VM Mutex/Monitor currently owned by a thread: None


[error occurred during error reporting (printing compressed oops mode), id 0xb, SIGSEGV (0xb) at pc=0x00007fd63bcfc543]

GC Heap History (0 events):
No events

Deoptimization events (0 events):
No events

Classes redefined (0 events):
No events

Internal exceptions (0 events):
No events

Events (2 events):
Event: 0.001 Protecting memory [0x00007fff3f2fe000,0x00007fff3f302000] with protection modes 0
Event: 0.002 Loaded shared library /opt/java/openjdk/lib/libzip.so


Dynamic libraries:
555c44800000-555c44801000 r-xp 00000000 ca:01 10815                      /opt/java/openjdk/bin/java
555c44a01000-555c44a02000 r--p 00001000 ca:01 10815                      /opt/java/openjdk/bin/java
555c44a02000-555c44a03000 rw-p 00002000 ca:01 10815                      /opt/java/openjdk/bin/java
555c45b51000-555c45bc4000 rw-p 00000000 00:00 0                          [heap]
7fd62506e000-7fd6252dd000 rw-p 00000000 00:00 0 
7fd6252dd000-7fd62554d000 rwxp 00000000 00:00 0 
7fd62554d000-7fd62586c000 ---p 00000000 00:00 0 
7fd62586c000-7fd625adc000 rwxp 00000000 00:00 0 
7fd625adc000-7fd62cda4000 ---p 00000000 00:00 0 
7fd62cda4000-7fd62d014000 rwxp 00000000 00:00 0 
7fd62d014000-7fd6342dd000 ---p 00000000 00:00 0 
7fd6342dd000-7fd639e00000 r--s 00000000 ca:01 410620                     /opt/java/openjdk/lib/modules
7fd639e00000-7fd639e07000 r-xp 00000000 ca:01 410619                     /opt/java/openjdk/lib/libzip.so
7fd639e07000-7fd63a007000 ---p 00007000 ca:01 410619                     /opt/java/openjdk/lib/libzip.so
7fd63a007000-7fd63a008000 r--p 00007000 ca:01 410619                     /opt/java/openjdk/lib/libzip.so
7fd63a008000-7fd63a009000 rw-p 00008000 ca:01 410619                     /opt/java/openjdk/lib/libzip.so
7fd63a0b2000-7fd63a200000 rw-p 00000000 00:00 0 
7fd63a200000-7fd63a21b000 r-xp 00000000 ca:01 410602                     /opt/java/openjdk/lib/libjimage.so
7fd63a21b000-7fd63a41a000 ---p 0001b000 ca:01 410602                     /opt/java/openjdk/lib/libjimage.so
7fd63a41a000-7fd63a41c000 r--p 0001a000 ca:01 410602                     /opt/java/openjdk/lib/libjimage.so
7fd63a41c000-7fd63a41d000 rw-p 0001c000 ca:01 410602                     /opt/java/openjdk/lib/libjimage.so
7fd63a4fe000-7fd63a4ff000 ---p 00000000 00:00 0 
7fd63a4ff000-7fd63a600000 rw-p 00000000 00:00 0 
7fd63a600000-7fd63a628000 r-xp 00000000 ca:01 410598                     /opt/java/openjdk/lib/libjava.so
7fd63a628000-7fd63a828000 ---p 00028000 ca:01 410598                     /opt/java/openjdk/lib/libjava.so
7fd63a828000-7fd63a829000 r--p 00028000 ca:01 410598                     /opt/java/openjdk/lib/libjava.so
7fd63a829000-7fd63a82a000 rw-p 00029000 ca:01 410598                     /opt/java/openjdk/lib/libjava.so
7fd63a82a000-7fd63a82b000 rw-p 00000000 00:00 0 
7fd63a915000-7fd63a91a000 rw-p 00000000 00:00 0 
7fd63a91a000-7fd63aa00000 ---p 00000000 00:00 0 
7fd63aa00000-7fd63aa0d000 r-xp 00000000 ca:01 410618                     /opt/java/openjdk/lib/libverify.so
7fd63aa0d000-7fd63ac0c000 ---p 0000d000 ca:01 410618                     /opt/java/openjdk/lib/libverify.so
7fd63ac0c000-7fd63ac0e000 r--p 0000c000 ca:01 410618                     /opt/java/openjdk/lib/libverify.so
7fd63ac0e000-7fd63ac0f000 rw-p 0000e000 ca:01 410618                     /opt/java/openjdk/lib/libverify.so
7fd63ac15000-7fd63ac1a000 rw-p 00000000 00:00 0 
7fd63ac1a000-7fd63ad00000 ---p 00000000 00:00 0 
7fd63ad00000-7fd63ae00000 rw-p 00000000 00:00 0 
7fd63ae00000-7fd63c070000 r-xp 00000000 ca:01 410631                     /opt/java/openjdk/lib/server/libjvm.so
7fd63c070000-7fd63c270000 ---p 01270000 ca:01 410631                     /opt/java/openjdk/lib/server/libjvm.so
7fd63c270000-7fd63c335000 r--p 01270000 ca:01 410631                     /opt/java/openjdk/lib/server/libjvm.so
7fd63c335000-7fd63c370000 rw-p 01335000 ca:01 410631                     /opt/java/openjdk/lib/server/libjvm.so
7fd63c370000-7fd63c3c7000 rw-p 00000000 00:00 0 
7fd63c3d8000-7fd63c400000 r--p 00000000 ca:01 1316373                    /usr/lib/x86_64-linux-gnu/libc.so.6
7fd63c400000-7fd63c595000 r-xp 00028000 ca:01 1316373                    /usr/lib/x86_64-linux-gnu/libc.so.6
7fd63c595000-7fd63c5ed000 r--p 001bd000 ca:01 1316373                    /usr/lib/x86_64-linux-gnu/libc.so.6
7fd63c5ed000-7fd63c5f1000 r--p 00214000 ca:01 1316373                    /usr/lib/x86_64-linux-gnu/libc.so.6
7fd63c5f1000-7fd63c5f3000 rw-p 00218000 ca:01 1316373                    /usr/lib/x86_64-linux-gnu/libc.so.6
7fd63c5f3000-7fd63c600000 rw-p 00000000 00:00 0 
7fd63c600000-7fd63c610000 r-xp 00000000 ca:01 410583                     /opt/java/openjdk/lib/jli/libjli.so
7fd63c610000-7fd63c80f000 ---p 00010000 ca:01 410583                     /opt/java/openjdk/lib/jli/libjli.so
7fd63c80f000-7fd63c810000 r--p 0000f000 ca:01 410583                     /opt/java/openjdk/lib/jli/libjli.so
7fd63c810000-7fd63c811000 rw-p 00010000 ca:01 410583                     /opt/java/openjdk/lib/jli/libjli.so
7fd63c8b9000-7fd63c8be000 rw-p 00000000 00:00 0 
7fd63c8be000-7fd63c8c5000 ---p 00000000 00:00 0 
7fd63c8c5000-7fd63c8cd000 rw-s 00000000 ca:01 1060368                    /tmp/hsperfdata_root/1
7fd63c8cd000-7fd63c8ce000 r--p 00000000 ca:01 1316470                    /usr/lib/x86_64-linux-gnu/librt.so.1
7fd63c8ce000-7fd63c8cf000 r-xp 00001000 ca:01 1316470                    /usr/lib/x86_64-linux-gnu/librt.so.1
7fd63c8cf000-7fd63c8d0000 r--p 00002000 ca:01 1316470                    /usr/lib/x86_64-linux-gnu/librt.so.1
7fd63c8d0000-7fd63c8d1000 r--p 00002000 ca:01 1316470                    /usr/lib/x86_64-linux-gnu/librt.so.1
7fd63c8d1000-7fd63c8d2000 rw-p 00003000 ca:01 1316470                    /usr/lib/x86_64-linux-gnu/librt.so.1
7fd63c8d2000-7fd63c8e0000 r--p 00000000 ca:01 1316425                    /usr/lib/x86_64-linux-gnu/libm.so.6
7fd63c8e0000-7fd63c95c000 r-xp 0000e000 ca:01 1316425                    /usr/lib/x86_64-linux-gnu/libm.so.6
7fd63c95c000-7fd63c9b7000 r--p 0008a000 ca:01 1316425                    /usr/lib/x86_64-linux-gnu/libm.so.6
7fd63c9b7000-7fd63c9b8000 r--p 000e4000 ca:01 1316425                    /usr/lib/x86_64-linux-gnu/libm.so.6
7fd63c9b8000-7fd63c9b9000 rw-p 000e5000 ca:01 1316425                    /usr/lib/x86_64-linux-gnu/libm.so.6
7fd63c9b9000-7fd63c9bd000 rw-p 00000000 00:00 0 
7fd63c9bd000-7fd63c9be000 r--p 00000000 ca:01 1316387                    /usr/lib/x86_64-linux-gnu/libdl.so.2
7fd63c9be000-7fd63c9bf000 r-xp 00001000 ca:01 1316387                    /usr/lib/x86_64-linux-gnu/libdl.so.2
7fd63c9bf000-7fd63c9c0000 r--p 00002000 ca:01 1316387                    /usr/lib/x86_64-linux-gnu/libdl.so.2
7fd63c9c0000-7fd63c9c1000 r--p 00002000 ca:01 1316387                    /usr/lib/x86_64-linux-gnu/libdl.so.2
7fd63c9c1000-7fd63c9c2000 rw-p 00003000 ca:01 1316387                    /usr/lib/x86_64-linux-gnu/libdl.so.2
7fd63c9c2000-7fd63c9c3000 r--p 00000000 ca:01 1316468                    /usr/lib/x86_64-linux-gnu/libpthread.so.0
7fd63c9c3000-7fd63c9c4000 r-xp 00001000 ca:01 1316468                    /usr/lib/x86_64-linux-gnu/libpthread.so.0
7fd63c9c4000-7fd63c9c5000 r--p 00002000 ca:01 1316468                    /usr/lib/x86_64-linux-gnu/libpthread.so.0
7fd63c9c5000-7fd63c9c6000 r--p 00002000 ca:01 1316468                    /usr/lib/x86_64-linux-gnu/libpthread.so.0
7fd63c9c6000-7fd63c9c7000 rw-p 00003000 ca:01 1316468                    /usr/lib/x86_64-linux-gnu/libpthread.so.0
7fd63c9c7000-7fd63c9c9000 r--p 00000000 ca:01 1316504                    /usr/lib/x86_64-linux-gnu/libz.so.1.2.11
7fd63c9c9000-7fd63c9da000 r-xp 00002000 ca:01 1316504                    /usr/lib/x86_64-linux-gnu/libz.so.1.2.11
7fd63c9da000-7fd63c9e0000 r--p 00013000 ca:01 1316504                    /usr/lib/x86_64-linux-gnu/libz.so.1.2.11
7fd63c9e0000-7fd63c9e1000 ---p 00019000 ca:01 1316504                    /usr/lib/x86_64-linux-gnu/libz.so.1.2.11
7fd63c9e1000-7fd63c9e2000 r--p 00019000 ca:01 1316504                    /usr/lib/x86_64-linux-gnu/libz.so.1.2.11
7fd63c9e2000-7fd63c9e3000 rw-p 0001a000 ca:01 1316504                    /usr/lib/x86_64-linux-gnu/libz.so.1.2.11
7fd63c9e3000-7fd63c9e4000 ---p 00000000 00:00 0 
7fd63c9e4000-7fd63c9e5000 r--p 00000000 00:00 0 
7fd63c9e5000-7fd63c9e7000 rw-p 00000000 00:00 0 
7fd63c9e7000-7fd63c9e9000 r--p 00000000 ca:01 1316355                    /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
7fd63c9e9000-7fd63ca13000 r-xp 00002000 ca:01 1316355                    /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
7fd63ca13000-7fd63ca1e000 r--p 0002c000 ca:01 1316355                    /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
7fd63ca1f000-7fd63ca21000 r--p 00037000 ca:01 1316355                    /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
7fd63ca21000-7fd63ca23000 rw-p 00039000 ca:01 1316355                    /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
7fff3f2fe000-7fff3f302000 ---p 00000000 00:00 0 
7fff3fadd000-7fff3fafe000 rw-p 00000000 00:00 0                          [stack]
7fff3fafe000-7fff3fb02000 r--p 00000000 00:00 0                          [vvar]
7fff3fb02000-7fff3fb04000 r-xp 00000000 00:00 0                          [vdso]


VM Arguments:
java_command: org.springframework.boot.loader.JarLauncher --spring.config.additional-location=file:/config/
java_class_path (initial): .
Launcher Type: SUN_STANDARD

[Global flags]
     intx CICompilerCount                          = 2                                         {product} {ergonomic}
     uint G1ConcRefinementThreads                  = 2                                         {product} {ergonomic}
   size_t G1HeapRegionSize                         = 1048576                                   {product} {ergonomic}
    uintx GCDrainStackTargetSize                   = 64                                        {product} {ergonomic}
   size_t InitialHeapSize                          = 98566144                                  {product} {ergonomic}
   size_t MaxHeapSize                              = 1551892480                                {product} {ergonomic}
   size_t MinHeapDeltaBytes                        = 1048576                                   {product} {ergonomic}
    uintx NonNMethodCodeHeapSize                   = 5825164                                {pd product} {ergonomic}
    uintx NonProfiledCodeHeapSize                  = 122916538                              {pd product} {ergonomic}
    uintx ProfiledCodeHeapSize                     = 122916538                              {pd product} {ergonomic}
    uintx ReservedCodeCacheSize                    = 251658240                              {pd product} {ergonomic}
     bool SegmentedCodeCache                       = true                                      {product} {ergonomic}
     bool UseCompressedClassPointers               = true                                 {lp64_product} {ergonomic}
     bool UseCompressedOops                        = true                                 {lp64_product} {ergonomic}
     bool UseG1GC                                  = true                                      {product} {ergonomic}

Logging:
Log output configuration:
 #0: stdout all=warning uptime,level,tags
 #1: stderr all=off uptime,level,tags

Environment Variables:
JAVA_HOME=/opt/java/openjdk
PATH=/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
LANG=en_US.UTF-8
LC_ALL=en_US.UTF-8

Signal Handlers:
SIGSEGV: [libjvm.so+0xf44160], sa_mask[0]=11111111011111111101111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGBUS: [libjvm.so+0xf44160], sa_mask[0]=11111111011111111101111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGFPE: [libjvm.so+0xf44160], sa_mask[0]=11111111011111111101111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGPIPE: [libjvm.so+0xc669b0], sa_mask[0]=11111111011111111101111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGXFSZ: [libjvm.so+0xc669b0], sa_mask[0]=11111111011111111101111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGILL: [libjvm.so+0xf44160], sa_mask[0]=11111111011111111101111111111110, sa_flags=SA_RESTART|SA_SIGINFO
SIGUSR2: [libjvm.so+0xc66850], sa_mask[0]=00000000000000000000000000000000, sa_flags=SA_RESTART|SA_SIGINFO
SIGHUP: SIG_DFL, sa_mask[0]=00000000000000000000000000000000, sa_flags=none
SIGINT: SIG_DFL, sa_mask[0]=00000000000000000000000000000000, sa_flags=none
SIGTERM: SIG_DFL, sa_mask[0]=00000000000000000000000000000000, sa_flags=none
SIGQUIT: SIG_DFL, sa_mask[0]=00000000000000000000000000000000, sa_flags=none


---------------  S Y S T E M  ---------------

OS:DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04 LTS"
uname:Linux 5.10.0-11-amd64 #1 SMP Debian 5.10.92-2 (2022-02-28) x86_64
OS uptime: 0 days 0:02 hours
libc:glibc 2.35 NPTL 2.35 
rlimit (soft/hard): STACK 8192k/infinity , CORE 0k/infinity , NPROC 4194304/4194304 , NOFILE 1048576/1048576 , AS infinity/infinity , CPU infinity/infinity , DATA infinity/infinity , FSIZE infinity/infinity , MEMLOCK 64k/64k
load average:0.06 0.06 0.02

/proc/meminfo:
MemTotal:        6059084 kB
MemFree:         4977548 kB
MemAvailable:    5338992 kB
Buffers:           47840 kB
Cached:           484988 kB
SwapCached:            0 kB
Active:           174920 kB
Inactive:         773296 kB
Active(anon):       5756 kB
Inactive(anon):   390696 kB
Active(file):     169164 kB
Inactive(file):   382600 kB
Unevictable:           8 kB
Mlocked:               8 kB
SwapTotal:        998396 kB
SwapFree:         998396 kB
Dirty:              1288 kB
Writeback:             0 kB
AnonPages:        413300 kB
Mapped:           275992 kB
Shmem:             15104 kB
KReclaimable:      39628 kB
Slab:              71456 kB
SReclaimable:      39628 kB
SUnreclaim:        31828 kB
KernelStack:        4592 kB
PageTables:         8656 kB
NFS_Unstable:          0 kB
Bounce:                0 kB
WritebackTmp:          0 kB
CommitLimit:     4027936 kB
Committed_AS:    1370008 kB
VmallocTotal:   34359738367 kB
VmallocUsed:       17580 kB
VmallocChunk:          0 kB
Percpu:            34048 kB
HardwareCorrupted:     0 kB
AnonHugePages:     45056 kB
ShmemHugePages:        0 kB
ShmemPmdMapped:        0 kB
FileHugePages:         0 kB
FilePmdMapped:         0 kB
HugePages_Total:       0
HugePages_Free:        0
HugePages_Rsvd:        0
HugePages_Surp:        0
Hugepagesize:       2048 kB
Hugetlb:               0 kB
DirectMap4k:      120832 kB
DirectMap2M:     3016704 kB
DirectMap1G:     4194304 kB

/sys/kernel/mm/transparent_hugepage/enabled:
[always] madvise never
/sys/kernel/mm/transparent_hugepage/defrag (defrag/compaction efforts parameter):
always defer defer+madvise [madvise] never

Process Memory:
Virtual Size: 384136K (peak: 384136K)
Resident Set Size: 17892K (peak: 17892K) (anon: 5484K, file: 12408K, shmem: 0K)
Swapped out: 0K
C-Heap outstanding allocations: 390K

/proc/sys/kernel/threads-max (system-wide limit on the number of threads):
47080
/proc/sys/vm/max_map_count (maximum number of memory map areas a process may have):
65530
/proc/sys/kernel/pid_max (system-wide limit on number of process identifiers):
4194304

Xen hardware-assisted virtualization detected
Steal ticks since vm start: 0
Steal ticks percentage since vm start:  0.000

CPU:total 2 (initial active 2) (12 cores per cpu, 1 threads per core) family 23 model 113 stepping 0 microcode 0x8701021, cmov, cx8, fxsr, mmx, sse, sse2, sse3, ssse3, sse4.1, sse4.2, popcnt, avx, avx2, aes, clmul, mmxext, lzcnt, sse4a, tsc, bmi1, bmi2, adx, sha, fma
CPU Model and flags from /proc/cpuinfo:
model name	: AMD Ryzen 5 3600 6-Core Processor
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm rep_good nopl cpuid extd_apicid tsc_known_freq pni pclmulqdq ssse3 fma cx16 sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm cr8_legacy abm sse4a misalignsse 3dnowprefetch bpext ssbd ibpb vmmcall fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt clwb sha_ni xsaveopt xsavec xgetbv1 xsaves clzero xsaveerptr arat umip rdpid
Online cpus:
0-1
Offline cpus:
2-63
BIOS frequency limitation:
<Not Available>
Frequency switch latency (ns):
<Not Available>
Available cpu frequencies:
<Not Available>
Current governor:
<Not Available>
Core performance/turbo boost:
<Not Available>

Memory: 4k page, physical 6059084k(4977548k free), swap 998396k(998396k free)

vm_info: OpenJDK 64-Bit Server VM (11.0.15+10) for linux-amd64 JRE (11.0.15+10), built on Apr 19 2022 21:36:55 by "" with gcc 7.5.0

END.
```
</p>
</details>

### Komga version

0.154.3

### Operating system

Debian 11

### Other details

#899 seems potentially related, though my container always crashes on startup without ever getting into the webinterface.  

Something must have changed between 0.154.1 and the later two versions to suddenly cause this issue. Reverting to version 0.154.1 works perfectly fine still.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/faq/).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
How do you run the container? Did you try recreating entirely?

**Comment by RealStickman** [comment]
This is the command I use to start the container:  
```
# podman run --name komga -p 8080:8080 \
    -v /mnt/configs/komga:/config \
    -v /mnt/books/komga:/content \
    --restart unless-stopped \
    -d docker.io/gotson/komga:latest
```
I did delete the container in between to try out the few latest versions until I found the last working one.

**Comment by gotson** [comment]
Care to try with docker instead of podman?

**Comment by gotson** [comment]
Seems the base image of Temurin was updated to Ubuntu jammy about a month ago.

Probably related, but still unsure why your system doesn't work.

Is that a container running in a vm?

**Comment by RealStickman** [comment]
The container is running inside a vm.  

Running the container with docker works.  
Using a new vm with podman also works (cloned from a template I have, so exact same base).  

Searching for `pthread_create failed (EPERM)` in relation to java and podman eventually lead me to [this issue on the temurin github](https://github.com/adoptium/temurin-build/issues/2974)  

Apparently my podman version was too old as well. Updating it to the latest version available makes the container work again.  

Thanks for your help, seems like I should update the base OS next time before opening an issue.  :)

**Comment by gotson** [comment]
Note for other people who may have the same issue:

Here are some more details on what's happening: https://github.com/adoptium/containers/issues/215#issuecomment-1142046045

To resolve the issue, update your docker-engine to 20.10.16.

The issue doesn't just affect Komga, it will affect a lot or other containers if you don't upgrade your docker.
